Data_Visualization_Workshop.Rmd

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
there should not be YAML header.

Reply:
It's fixed now. Please check @skyetim

Reply:
I have removed the YAML header please check